### DISCUSSION TITLE
Feature request: add shape selection settings

### DISCUSSION THREAD

**Discussion by Manueljlin** [uxbox/uxbox#527] [Ideas]
This allows to combine, split, substract, intersect and flatten shapes

Figma:
![image](https://user-images.githubusercontent.com/11275573/106603506-a97ffa80-655e-11eb-992b-696392f52da5.png)

Adobe Illustrator:
![image](https://user-images.githubusercontent.com/11275573/106603565-bb619d80-655e-11eb-9ff6-5ff99809f33e.png)

**Comment by elhombretecla** [comment]
Hi Manuel!!! Thank you for your feedback :)
We have boolean operations between shapes and SVG elements in our roadmap.
It is an important feature to work with vector shapes and create more advanced designs. We will start working on this soon!
